controllers/wishlist.controllers.js

```diff
@@ -49,9 +50,7 @@ async function getAllWishlistItems(req, res, next) {
     return next(error);
   }
 
-  return res.status(200).json({
-    ...wishlistItems,
-  });
+  return res.status(200).json(...wishlistItem);
```

Comment:
move res.status(200)... into try block and declare wishlistItem variable in try block too, using const keyword

Reply:
done